synergycp.php

```diff
@@ -4,7 +4,7 @@
     die('This file cannot be accessed directly.');
 }
 
-ini_set('display_errors', 'Off');
+ini_set('display_errors', 'On');
```

Comment:
shouldn't change this on the production version

Reply:
i just have this set in mine, i'll remove it before the final PR